federation/manifests/federation-apiserver-deployment.yaml

```diff
@@ -23,7 +23,6 @@ spec:
         - --etcd-servers=http://localhost:2379
         - --service-cluster-ip-range={{.FEDERATION_SERVICE_CIDR}}
         - --secure-port=443
-        - --advertise-address={{.FEDERATION_API_HOST}}
```

Comment:
> What existing behavior are we breaking? As far as I know, there is no functionality in the federation control plane which is affected by the --advertise-address 

There is, apparently, and I mentioned it in this code review earlier. We do use this field for doc generation.

> simply defaulting to the bind address. 

And defaulting to bind address is not even correct. If you have 2 pods for a federation API server, what's the right address to advertise?

> If there were, we should fix it so it will work on all platforms, rather than just GCE/local dev (rather than make this change here).

There is no second question about that. But that doesn't mean we need to unnecessarily change this.